bigtable/client/row_test.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/row.h"
+
+#include <gtest/gtest.h>
+
+/// @test Verify Row instantiation and trivial accessors.
+TEST(RowTest, Simple) {
```

Comment:
You should have a test with more than one cell, and a test with 0 cells, though they will obviously work.

Reply:
done